### DISCUSSION TITLE
Volo.Abp.Sms.Twilio module add

### DISCUSSION THREAD

**Discussion by recruitans** [abpframework/abp#7426] [Q&A]
Why does Volo.Abp.Sms.Twilio's under folders module not appear in Solution Explorer?
I added it with from abp suite.

![image](https://user-images.githubusercontent.com/28866278/105520661-08867980-5cec-11eb-8a10-fb8c9d0c18e4.png)

**Comment by maliming** [comment]
hi

You can get help on https://support.abp.io/ website.
